shard-operator-parent/shard-operator-v1/src/main/java/com/redhat/service/smartevents/shard/operator/v1/resources/istio/virtualservice/VirtualService.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.service.smartevents.shard.operator.resources.istio.virtualservice;
+package com.redhat.service.smartevents.shard.operator.v1.resources.istio.virtualservice;
```

Comment:
`virtualservice` could be moved to `core`

Reply:
Done. I've moved `knative` and `istio` to `-core`.